README.md

```diff
@@ -1,4 +1,14 @@
-# Wave Recovery Tool
+# Wave Recovery Tool GUI
```

Comment:
Please restore the original heading `Wave Recovery Tool`.

Then please move these two new sections between `Authors` and `Prerequisites`. Maybe we could also rename the section `Wave Recovery Tool GUI` to `Graphical User Interface`?